release/pkg/file_reader.go

```diff
@@ -30,16 +34,30 @@ import (
 
 var imageBuilderProjectSource = "projects/kubernetes-sigs/image-builder"
 
-func (r *ReleaseConfig) readShaSums(filename string) (sha256, sha512 string, err error) {
-	sha256Path := filename + ".sha256"
-	sha256, err = readShaFile(sha256Path)
-	if err != nil {
-		return "", "", errors.Cause(err)
-	}
-	sha512Path := filename + ".sha512"
-	sha512, err = readShaFile(sha512Path)
-	if err != nil {
-		return "", "", errors.Cause(err)
+func (r *ReleaseConfig) readShaSums(filename string) (string, string, error) {
+	var sha256, sha512 string
+	var err error
+	if r.DryRun {
+		sha256, err = GenerateRandomSha(256)
```

Comment:
dumb question: why do we need to fake the sha?
Shouldn't we be able to compute it even if we don't reupload the artifacts?

Reply:
as below, the dry run is run either locally or in the Prow presubmit which is in a different account than the EKS-A build account, so it cannot call the ECR APIs pertaining to the build account.